homework/thinhbka/lesson-4-hashing/2_Maximum_Subarray.py

```diff
@@ -0,0 +1,69 @@
+# arr  = [-1, 4, -1, 7, 8]
+
+#sum(i,j) = preS[j] - preS[i-1] 
+# sum(i,j) max <=> preS[j] is biggest, and preS[i-1] smallest, i-1 <= j
+class Solution:
+    def maxSubArray(self, nums: List[int]) -> int:
+        preS =[0]
```

Comment:
How about this one @ThinhNgVhust, I think the variable's name should be `pre_sum`